ubuntucleaner/utils/icon.py

```diff
@@ -70,8 +70,6 @@ def get_from_mime_type(mime, size=DEFAULT_SIZE):
         log.error('get_from_mime_type failed: %s' % e)
         return get_from_name(size=size)
 
-    return pixbuf
```

Comment:
Looked like dead code to me, unless I missed some magic?

Reply:
indeed you are totally right here, makes no sense that statement